sc2ts/utils.py

```diff
@@ -513,156 +718,124 @@ def recombinants_summary(self):
-        ) - 1
-
-        causal_sample_map = get_recombinant_samples(self.ts)
-        tree_a = self.ts.first()    
-        tree_b = self.ts.first()
-        node_times = self.ts.nodes_time
+        recombs = self.combine_recombinant_info()
         data = []
-        for i_a, i_b, brk in zip(tree_a_indexes, tree_b_indexes, sorted_breakpoints):
-            tree_a.seek_index(i_a)
-            tree_b.seek_index(i_b)
-            for nd, hmm_cons, (l_parent, r_parent) in forward_breakpoints[brk]:
-                num_nodes_in_path = 0
-                node_a = l_parent
-                node_b = r_parent
-                row = {
-                    "in_arg": tree_a.parent(nd) == node_a and tree_b.parent(nd) == node_b
-                }
-                while True:
-                    if node_a == tskit.NULL or node_b == tskit.NULL:
-                        break
-                    if node_times[node_a] == node_times[node_b]:
-                        if node_a == node_b:
-                            break
-                        else:
-                            if tree_a.parent(node_a) < tree_b.parent(node_b):
-                                node_a = tree_a.parent(node_a)
-                            else:
-                                node_b = tree_b.parent(node_b)
-                            num_nodes_in_path += 1
-                                
-                    elif node_times[node_a] < node_times[node_b]:
-                        node_a = tree_a.parent(node_a)
-                        num_nodes_in_path += 1
-                    elif node_times[node_b] < node_times[node_a]:
-                        node_b = tree_b.parent(node_b)
-                        num_nodes_in_path += 1
-                mrca = node_a
-                row["node"] = nd
-                row["max_descendant_samples"] = self.nodes_max_descendant_samples[nd]
-                row["breakpoint"] = brk
-                row["hmm_consistent"] = hmm_cons
-                row["left_parent"] = l_parent
-                row["left_parent_pango"] = meta[l_parent].get("Imputed_lineage", "")
-                row["right_parent"] = r_parent
-                row["right_parent_pango"] = meta[r_parent].get("Imputed_lineage", "")
-                row["parents_mrca"] = mrca
-                row["tmrca"] = np.nan
-                row["tmrca_delta"] = np.nan
-                row["nodes_between_parents"] = np.nan
-                # Pick one of the causal strains and report its Nextclade label
-                row["origin_nextclade_pango"] = meta[causal_sample_map[nd]]["Nextclade_pango"]
-
-                if mrca != tskit.NULL:
-                     row["tmrca"] = node_times[mrca]
-                     row["tmrca_delta"] = node_times[mrca] - node_times[nd]
-                     row["nodes_between_parents"] = num_nodes_in_path
+        for rec in recombs:
+            arg = rec.arg_info
+            for j in range(len(arg.parents) - 1):
+                row = rec.data_summary()
```

Comment:
Does this include the time of the recombination node? I don't think it does, and it would be helpful in this csv output so that I could calculate the difference between the re node time and the mrca time?